### PR TITLE
Add BJ's Wholesale

### PR DESCRIPTION
**Pull request by tas50**
A Wholesale club on the east coast of the US.

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: config/canonical.json [hunk 1]
```diff
@@ -34666,6 +34666,17 @@
       "shop": "watches"
     }
   },
+  "shop/wholesale|BJ's Wholesale Club": {
+    "nocount": true,
+    "tags": {
+      "brand": "BJ's Wholesale Club",
+      "brand:wikidata": "Q4835754",
+      "brand:wikipedia": "en:BJ's Wholesale Club",
```
Comment (bhousel) [inline config/canonical.json]: The `brand:wikipedia` tag needs to have underscores in it, like `"en:BJ's_Wholesale_Club"`.   
Can you change these on any of your open PRs?

**Comment by tas50** [comment]
@bhousel Can you confirm which format the Wikipedia entries should be. npm run build wants them without underscores. Adding the underscores results in this warning:

```
Warning - Entries in `canonical.json` with incorrect `brand:wikipedia` format:
To resolve these, make sure "brand:wikipedia" tag looks like "en:Pizza Hut".
  "shop/wholesale|BJ's Wholesale Club" -> "brand:wikipedia": "en:BJ's_Wholesale_Club"
```

**Comment by bhousel** [comment]
> @bhousel Can you confirm which format the Wikipedia entries should be. npm run build wants them without underscores. Adding the underscores results in this warning:

Yea, sorry I was wrong about the underscores.. it should be no underscores..

File: config/canonical.json (hunk 1 above)
Comment (bhousel) [reply config/canonical.json]: Sorry, what I wrote here was wrong. 😆

**Pull request merged**